docs/Performance/promise.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: default
+title: Promise
+name: Overview
+parent: Performance
+nav_order: 1
+---
+
+## Promise
+
+If you are new to Promises, I would suggest to make yourself familiar before continuing:
+
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise>
+
+### Basic OData call
+
+This is a very basic OData call where the key and the callbacks are hardcoded.
+
+```js
+var model = that.getView().getModel();
+model.read("/MyEntitySet('12345')", {
+  success: function (data, response) {
+    // success handler
+  },
+  error: function (error) {
+    // error handler
+  }
+});
+```
+
+### Promisified OData call
+
+A wrapped version of the basic OData call.
+
+The key is given as parameter and the wrapping method returns a promise. The resolve / reject methods are used in the success / error callbacks to resolve or reject the Promise.
```

Comment:
code fence `resolve`/`reject`, `success` / `error` - and please do the same throughout the file

Reply:
Changed with the latest commit